elfi/methods/posteriors.py

```diff
@@ -58,13 +58,13 @@ def __init__(self, model, threshold=None, prior=None, n_inits=10, max_opt_iters=
 
         if self.threshold is None:
             # TODO: the evidence could be used for a good guess for starting locations
-            minloc, minval = minimize(self.model.predict_mean,
-                                      self.model.bounds,
-                                      self.model.predictive_gradient_mean,
-                                      self.prior,
-                                      self.n_inits,
-                                      self.max_opt_iters,
-                                      random_state=self.random_state)
+            minloc, minval = bo.minimize(self.model.predict_mean,
```

Comment:
This gives the impression that `minimize` uses Bayesian optimization, which is not the case. (But yeah, the whole function could be moved out of BO.)